src/theory/ext_theory.cpp

```diff
@@ -237,7 +237,7 @@ bool ExtTheory::doInferencesInternal(int effort,
           if (!nr.isNull() && n != nr)
           {
             Node lem = NodeManager::currentNM()->mkNode(kind::EQUAL, n, nr);
-            if (sendLemma(lem, true))
+            if (sendLemma(lem, InferenceId::UNKNOWN, true))
```

Comment:
Can you add an InferenceId for this?  I'd suggest the name `EXTT_SIMPLIFY`.

Reply:
Done